spikeinterface/widgets/matplotlib/spikes_on_traces.py

```diff
@@ -47,6 +47,8 @@ def do_plot(self, data_plot, **backend_kwargs):
             spike_frames = sorting.get_unit_spike_train(unit, segment_index=segment_index)
             spike_start, spike_end = np.searchsorted(spike_frames, frame_range)
             
+            chan_inds = dp.sparsity.unit_id_to_channel_indices[unit]
```

Comment:
I think this should be:
```suggestion
            chan_ids = dp.sparsity.unit_id_to_channel_ids[unit]
```

since at L83 we loop over `channel_ids`, and not `channel_indices`.

No?